src/test/java/com/google/api/codegen/metacode/SampleInitCodeTest.java

```diff
@@ -350,92 +369,62 @@ public void testMultipleFormattedEntries() throws Exception {
         Arrays.asList(
             "formatted_field%entity1", "formatted_field%entity2", "formatted_field%entity3");
 
-    List<String> expectedKeyList = Arrays.asList("formatted_field", "root");
-
-    HashMap<String, InitValueConfig> initValueMap = new HashMap<>();
     InitValueConfig initValueConfig = InitValueConfig.create("test-api", null);
-    initValueMap.put("formatted_field", initValueConfig);
 
     InitCodeContext context =
         getContextBuilder()
             .initFieldConfigStrings(fieldSpecs)
-            .initValueConfigMap(initValueMap)
+            .initValueConfigMap(ImmutableMap.of("formatted_field", initValueConfig))
             .build();
     InitCodeNode rootNode = InitCodeNode.createTree(context);
-    List<String> actualKeyList = new ArrayList<>();
-    for (InitCodeNode node : rootNode.listInInitializationOrder()) {
-      actualKeyList.add(node.getKey());
-    }
-    Truth.assertThat(actualKeyList.equals(expectedKeyList)).isTrue();
+    assertThat(rootNode.listInInitializationOrder().stream().map(node -> node.getKey()))
+        .containsExactly("formatted_field", "root")
+        .inOrder();
```

Comment:
Can we test `entity1`, `entity2`?

Reply:
I don't think we can. The `entity1` and `entity2` will be missing from the data structure for the same reason as [here](https://github.com/googleapis/gapic-generator/pull/2178#discussion_r208031764). I don't really know what the reason is.

Reply:
Urgh.... I don't like mysterious code.